src/Orc.Theming/Behaviors/Enums/PositionCorner.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Orc.Theming
+{
+    public enum PositionCorner
```

Comment:
I don't like the name of this class. I think we should stick with the WPF "standards" and use HorizontalAlignment and VerticalAlignment.

Reply:
Yes, the naming doesn't really fit in WPF "standard", but it feels like HorizontalAlignment/VerticalAlignment are defined for i bit different purpose, as they align element against **parent**, while adorner isn't really child element of textbox. It's more simillar to behavior of tooltip or something else like that. I don't like aligment also because it has 'stretch' position which doesn't make sense for me for adorner.
### Suggestions
We can change the name to _PlacementMode_ or _AdornerPlacementMode_ (to make it more simillar to other wpf parts which are drawn on top of UIElements) or i found the name used in old good WinForms for enum which was used for simillar purposes - _'ArrangePosition'_